stable/chartmuseum/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Helm Chart Repository with support for Amazon S3 and Google Cloud Storage
 name: chartmuseum
-version: 1.4.0
+version: 1.4.1
```

Comment:
Another PR has been merged since. Also, this is technically a new feature. Can you switch this to `1.6.0`?

Reply:
Done